server/src/index.js

```diff
@@ -83,8 +83,8 @@ function normalizeEmoji(emoji) {
   return res
 }
 
-app.get('/api/reports-by-tags/:tag', authorize, async (req, res) => {
-  let reports = await loadReportsByTag(req.params.tag)
+async function loadReportData() {
+  let reports = await loadReports()
```

Comment:
There might be some overlap between reports, but I am assuming it will be minimal and the overlap itself is handled quite well because of our cache (it is not loaded two times).